site/src/components/MultiLanguageCode.js

```diff
@@ -8,6 +8,7 @@ const classNameToLanguage = {
     "language-c3": "c3",
     "language-cpp": "c",
     "language-d": "d",
+    "language-grain": "grain",
```

Comment:
I don't even know what this file is, but I added grain alongside the other languages.